specification/metrics/negotiatedunitprice.md

```diff
@@ -0,0 +1,31 @@
+# Negotiated Unit Price
+
+The Negotiated Unit Price represents a unit price for a single pricing measurement unit, which incorporates volume/tier-based rates and negotiated discounts while excluding any commitment-based discounts or the amortization of upfront charges (one-time or recurring). This price is denominated in the  [Billing Currency](#billingcurrency). The Negotiated Unit Price equals the [List Unit Price](#listunitprice) when no negotiated discounts are present. It is used to calculate [Negotiated Cost](#negotiatedcost) and savings based on negotiation activities.
```

Comment:
To include the capability to calculate savings from utilizing spare capacity (spot) discounts, we are still exploring a couple of alternative Negotiated Unit Price definitions.

- OPTION A (current version): _The Negotiated Unit Price represents a unit price for a single pricing measurement unit, which incorporates volume/tier-based rates and negotiated discounts while excluding any commitment-based discounts or the amortization of upfront charges (one-time or recurring)._

- OPTION B : _The Negotiated Unit Price represents a unit price for a single pricing measurement unit, which incorporates volume/tier-based rates and negotiated discounts while excluding any commitment-based discounts, **spare capacity (spot) discounts** or the amortization of upfront charges (one-time or recurring)._

- OPTION C (aligned with pricing model column terminology): _The Negotiated Unit Price represents a unit price for a single pricing measurement unit, which incorporates volume/tier-based rates and negotiated discounts while excluding any commitment-based discounts, **the impacts of dynamic pricing model** or the amortization of upfront charges (one-time or recurring)._

Note: **Alignment of all unit price and cost metric definitions is implied.**

Reply:
I like option C better. I would rephrase it for this: 

The Negotiated Unit Price represents a unit price for a single pricing measurement unit **agreed between the Cloud Service Provider and the Service Receiver**. Negotiated Unit Price incorporates volume/tier-based rates and negotiated discounts while excluding any commitment-based discounts, the impacts of dynamic pricing model or the amortization of upfront charges (one-time or recurring).

Reply:
I like B:)

Reply:
I prefer option A (sorry!).

Indeed, I see spot (/ dynamic / interruptible) instances as separated SKUs than the regular on-demand SKUs, as the SLA are different: a spot instance can be interrupted. To me, spot instances are more low-cost than discounted. Thus, the negotiated cost of a spot should be the same as the billed cost.